code/espurna/scheduler.cpp

```diff
@@ -311,7 +337,7 @@ void schSetup() {
 
         static bool restore_once = true;
         if (restore_once) {
-            for (unsigned char i = 0; i < relayCount(); i++) {
+            for (unsigned char i = 0; i < schedulableCount(); i++) {
```

Comment:
Doesn't this step on schCheck id resolution? If we have 1 relay and 1 curtain, for example.

Reply:
I will check that, it searched a lot before doing that

Reply:
In this specific block, schCheck will recursively check every 'rule' that uses specific relayID (index from old relayCount()) and goes back a week day-by-day to check if rule applies in the past.
Although, we don't count light channels, so it's still sort-of broken because of non-matching ID / indexes.

Reply:
Understood! In fact, we need specific processing for curtain.